.cirrus.yml

```diff
@@ -143,6 +143,7 @@ smoke_task:
 build_task:
     alias: 'build'
     name: 'Build for $DISTRO_NV'
+    only_if: $CIRRUS_CHANGE_TITLE !=~ '.*CI:DOCS.*'
```

Comment:
Would you consider making this a macro, e.g. `&not_docs`, for readability (and DRY) below?

Reply:
good idea